packages/react-components-pixi-7/package.json

```diff
@@ -41,53 +41,49 @@
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@pixi/react-fiber": "*",
-    "@pixi/react-invariant": "*",
-    "@pixi/react-modular": "*",
```

Comment:
It's definitely helpful to alphabetize, but it might be useful to keep these at the top so it's easier to keep track of the inter monorepo deps?